Update Dhruv-Mathur.md - updated intro with better and language and more info (#1161)
SSWConsulting/SSW.People.Profiles: Dhruv-Mathur/Dhruv-Mathur.md

[[imgBadge]]
| ![.NET Core](../badges/Developer-dotnet-core.png)

> Dhruv Mathur is our go-to software developer here at SSW, especially when it comes to anything .NET MVC, .NET Web APIs, ASP.NET, and .NET Core. He's got a real talent for piecing together applications that are not just powerful, but also user-friendly. Thanks to his solid skills with Angular, React, and the essentials like CSS, Bootstrap, Tailwind, and HTML5, the apps he works on are easy on the eyes and a breeze to use.

Reviewer: ```suggestion
Dhruv Mathur is a superstar developer, especially when it comes to anything .NET MVC, .NET Web APIs, ASP.NET, and .NET Core. He's got a real talent for piecing together applications that are not just powerful, but also user-friendly. Thanks to his solid skills with Angular, React, and the essentials like CSS, Bootstrap, Tailwind, and HTML5, the apps he works on are easy on the eyes and a breeze to use.
```

> Dhruv Mathur is our go-to software developer here at SSW, especially when it comes to anything .NET MVC, .NET Web APIs, ASP.NET, and .NET Core. He's got a real talent for piecing together applications that are not just powerful, but also user-friendly. Thanks to his solid skills with Angular, React, and the essentials like CSS, Bootstrap, Tailwind, and HTML5, the apps he works on are easy on the eyes and a breeze to use.

Reviewer 2: ```suggestion
Dhruv is a full stack dev who specializes in .NET MVC, .NET Web APIs, ASP.NET, and .NET Core. He focuses on piecing together applications that are not just powerful, but also user-friendly. Thanks to his solid skills with Angular, React, and the essentials like CSS, Bootstrap, Tailwind, and HTML5, the apps he works on are easy on the eyes and a breeze to use.
```

1. Removed - SSW doesn't have a go to developer at all 😂
2. Making it sound less casual

> But there's more to Dhruv than just making things look good and work well. He's also big on keeping them secure, which is super important. He knows the ins and outs of application security, especially with IdentityServer, making sure that the data and privacy in the apps he builds are locked down tight.

Reviewer 2: ```suggestion
But there's more to Dhruv than just making things look good and work well. He knows the ins and outs of application security, especially with IdentityServer, making sure that the data and privacy in the apps he builds are locked down tight.
```

* removing the casual sentence

> Around SSW, Dhruv is known for constantly leveling up his skills in an environment that's all about learning and growing. He's the kind of guy who makes sure the technical stuff is spot on and that everyone's on the same wavelength. When Dhruv's on your project, you're not just getting an ace developer; you're getting a team player who's committed to nailing the project and making sure everyone walks away happy.

Reviewer 2: ```suggestion
Dhruv is constantly leveling up his skills while being a team leader. He makes sure the technical stuff is spot on, while ensuring that everyone is on the same wavelength. When Dhruv's on your project, you're not just getting an ace developer; you're getting a team player who's committed to nailing the project and making sure everyone walks away happy.
```

> ## Internal Projects

Reviewer: ```suggestion
## Projects
```
